keyboards/handwired/8key/rules.mk

```diff
@@ -0,0 +1,19 @@
+# MCU name
+MCU = atmega32u4
```

Comment:
All of this can be moved into the info.json.

Reply:
do you have an example info.json I can peek for this and the other comment about the json?

Reply:
@drashna do you have an example info.json I can peek for this and the other comment about the json?

I glanced around but couldnt find exactly whats needed to put in info.json for this.

Reply:
This would be a better example, stuff like "development_board" also sets the bootlaoder, controller, etc.

https://github.com/qmk/qmk_firmware/blob/f3d52d8fe81da5044db4bb675ae788393d7c06a5/keyboards/boardsource/holiday/spooky/info.json#L14